src/main/webapp/app/shared/connection-status/connection-status.component.scss

```diff
@@ -1,5 +1,5 @@
 .connection-status {
-    font-size: 0.85rem;
+    font-size: 0.75rem;
```

Comment:
As it appears that the font size shall be consistent, would it make sense to define a shared variable for the `font-size`?

Reply:
Yes, we could do this in a separate PR for multiple occurrences